balancer-js/src/modules/joins/joins.module.integration.spec.ts

```diff
@@ -274,21 +288,21 @@ describe('generalised join execution', async () => {
       //   authorisation: authorisation,
       //   wrapMainTokens: false,
       // },
-      {
-        signer,
-        description: 'join with 1 leaf and 1 linear',
-        pool: {
-          id: addresses.bbamaiweth.id,
-          address: addresses.bbamaiweth.address,
-        },
-        tokensIn: [addresses.WETH.address, addresses.bbamai.address],
-        amountsIn: [
-          parseFixed('10', 18).toString(),
-          parseFixed('10', 18).toString(),
-        ],
-        authorisation: authorisation,
-        wrapMainTokens: false,
-      },
+      // {
```

Comment:
should we add these tests back in and make sure they pass?

Reply:
I believe uncommented tests are comprehensive enough. The ones commented out are always a subset of an uncommented test. My suggestion is that we remove these redundant tests.
Ps: I uncommented and ran all of them just in case and they all passed ✅

Reply:
Lets remove the commented tests now if you're happy to.